Zend/Optimizer/zend_optimizer.c

```diff
@@ -831,6 +831,49 @@ zend_class_entry *zend_optimizer_get_class_entry_from_op1(
 	return NULL;
 }
 
+const zend_class_constant *zend_fetch_class_const_info(
+	const zend_script *script, const zend_op_array *op_array, const zend_op *opline, bool *is_prototype) {
+	const zend_class_entry *ce = NULL;
+	bool is_static_reference = false;
+
+	if (!opline || !op_array || !script || opline->op2_type != IS_CONST || Z_TYPE(ZEND_OP2_LITERAL(opline)) != IS_STRING) {
```

Comment:
Most code uses CRT_CONSTANT over ZEND_OPx_LITERAL, so let's use that for consistency. It works whether pass_two was done or not, although I think it is always undone here.